target/mips/translate_init.inc.c

```diff
@@ -549,10 +550,11 @@ const mips_def_t mips_defs[] =
         .SEGBITS = 40,
 #if defined(TARGET_CHERI)
         .PABITS = 40,
+        .insn_flags = CPU_MIPS3 | CPU_MIPS64R2 | ASE_MT,
```

Comment:
I don't think we should be adding MIPS64R2 here.
Generally we should be using the BERI cpu anyway.

Reply:
I think I added the flags I needed to both purely because I was not sure which CPU was actually the one in use. Is this the defunct one?

Reply:
Whether it's a useful one to use still or not, it's not appropriate for SMP. The R4K was single-core MIPS III. The optional MTE ASE only came about with MIPS 32/64 Release 2. If you want SMP use a different core that actually supports it.

Reply:
We currently similarly abuse the 5Kf (and some of that is my fault from the past). We should probably revert that and move everything over to the BERI CPU.

Reply:
Which cores do we use for which projects by default?

Reply:
Beri should be the default for target_cheri. Or maybe it's cheribuild that passes the flags to select beri